srml/democracy/src/lib.rs

```diff
@@ -101,6 +101,7 @@ decl_module! {
 		fn vote(origin, ref_index: Compact<ReferendumIndex>, approve_proposal: bool) -> Result {
 			let who = ensure_signed(origin)?;
 			let ref_index = ref_index.into();
+			// TODO - should we only allow voting after we also check that the data mapped to the ref_index in `ReferendumInfoOf` is valid
```

Comment:
Please make an github issue and put a link into the comment. We want each `TODO` to be reflected in the issue tracker.

Reply:
Sure, I've created this issue https://github.com/paritytech/substrate/issues/1173

Reply:
Thanks for adding the issue. 
Please also update the comment in the code with a link to the github issue. (I did not express this before very well.)